### PR TITLE
Test class/rub 191 196

### PR DESCRIPTION
**Pull request by sunilspatil**
Hello @RemarioCRM 
Please see the required test class files

### CODE REVIEW

File: force-app/main/default/classes/ContentDocTriggerHandler.cls [hunk 1]
```diff
@@ -0,0 +1,177 @@
+public class ContentDocTriggerHandler {
+    
+    public static void CheckIfAllDocumentsAreUploaded(list<ContentDocument> contenDocsList){
```
Comment (RemarioCRM) [inline force-app/main/default/classes/ContentDocTriggerHandler.cls:3]: @sunilspatil not seeing the latest changes for preassessment checkbox

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/ContentDocTriggerHandler.cls (hunk 1 above)
Comment (RemarioCRM) [reply force-app/main/default/classes/ContentDocTriggerHandler.cls:3]: merge with the release branch then push again

Comment (sunilspatil) [reply force-app/main/default/classes/ContentDocTriggerHandler.cls:3]: ok

**Comment by sunilspatil** [comment]
Hello @RemarioCRM 
I think we can ignore this branch as below files are already committed and merged with Rub 229.
      force-app/main/default/classes/ContentDocTriggerHandler.cls
      force-app/main/default/classes/ContentDocTriggerHandler.cls-meta.xml

And below file is already committed and merged when I created PR for PDF changes
      force-app/main/default/classes/CreditApplicationSummaryDocument.cls

Now only question remained is for the below file. I will create a new branch for this TestDataFactory.cls file and create PR.
     force-app/main/default/classes/TestDataFactory.cls

 

cc: @ishwari

**Pull request closed without merging**